.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+name: Check Markdown and Examples
+on:
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-python@v2.2.2
+      - uses: actions/checkout@v2
+      - run: pip install poetry
+      - run: poetry install
+      - run: npm install
+      - run: poetry run pre-commit run --all-files
```

Comment:
we are installing python to run `pre-commit` which then just runs the node apps? 

we can just call them directly

Reply:
`pre-commit` is an easy way to verify that everything is valid at commit time. I'd rather users didn't have to run `npm test` before every commit manually.

Reply:
You could just use a JS alternative eg https://github.com/typicode/husky

Reply:
if you just want pre-commit, `husky` is a javascript version that would do the exact same without having to install a 2nd language/package manager

Reply:
It looks like that is just a razor-thin wrapper around `git config core.hooksPath` (let me know if there's more to it). I could just use that, but we'd lose one of the main advantages of `pre-commit`, that it only lints the stuff that's staged to be committed (`stac-node-validator` is really slow).